requirements.txt

```diff
@@ -1,11 +1,12 @@
+black==18.6b4
```

Comment:
I like the idea, that I copied from some other codebase that I can't remember, of separating our dev dependencies in this file, like [in docs.snapcraft.io](https://github.com/canonical-websites/docs.snapcraft.io/blob/80c96f7658938bc9c9776208b1d3f05f58d32364/requirements.txt):

```
talisker==0.11.0
responses==0.10.4

# Development dependencies
black
coverage
```